examples/cli/main.go

```diff
@@ -57,6 +57,7 @@ func main() {
 				Destination: &echo,
 			}
 		}),
+		cli.Pprof(nil, ":6061"),
```

Comment:
may be it's better put it into constant or config?

Reply:
@kpeo this is just an example..